packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/WVERC20Details/WVERC20Details.tsx

```diff
@@ -96,10 +100,20 @@ const WVERC20Details = ({ onStepChange, onCreateTopic }: WVConsentProps) => {
         setTokenValue={setTokenValue}
         tokenValue={tokenValue}
         containerClassName="token-input"
-        disabled={editMode}
+        disabled={editMode || tokenValue == ZERO_ADDRESS}
         fullWidth
         tokenError={getTokenError()}
       />
+      <CWCheckbox
```

Comment:
shouldnt this be only visible (and logic applied) on particular chains 

per ticket:

 > If community is on Base or ETH, there will be a checkbox field beneath the token address field. This field would say "use native token"

Reply:
since we now have multiple chains supported, then I think this is supposed to apply to all of them